File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/firebase/database/FirebaseDatabaseActivity.java
package com.jdroid.android.sample.ui.firebase.database;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class FirebaseDatabaseActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return FirebaseDatabaseFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/firebase/database/FirebaseDatabaseActivity.kt
package com.jdroid.android.sample.ui.firebase.database

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class FirebaseDatabaseActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return FirebaseDatabaseFragment::class.java
    }
}
